coderd/audit.go

```diff
@@ -204,7 +204,6 @@ func (api *API) convertAuditLog(ctx context.Context, dblog database.GetAuditLogs
 			Deleted:            dblog.UserDeleted.Bool,
 			LastSeenAt:         dblog.UserLastSeenAt.Time,
 			QuietHoursSchedule: dblog.UserQuietHoursSchedule.String,
-			ThemePreference:    dblog.UserThemePreference.String,
```

Comment:
If I'm reading this correctly, this will cause folks looking at user change audit logs to no longer see previous change to `UserThemePreference`.

I don't think that's a huge deal in itself, but I could see a user considering it problematic.

If we take this same approach for something less innocuous, we'll probably run into the same issue without rewriting the audit logs (which I'd like to avoid).

Ideally we'd check for the legacy value in the audit log.

Also, do we need to audit changes to the `UserSettings`? For theme preference I would not consider it critical, but maybe there are other user-scoped settings we'll end up wanting to audit in there.

Reply:
theme preference just means "does the user use the light theme or the dark theme?", I think anyone managing a deployment who _really_ cares about that might be taking their job too seriously 😅